src/Instrumentation/MongoDB/README.md

```diff
@@ -0,0 +1,63 @@
+# OpenTelemetry MongoDB auto-instrumentation
+
+**Preferred and simplest way to install auto-instrumentation (c extension plus instrumentation libraries) is to use [opentelemetry-instrumentation-installer](https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/src/AutoInstrumentationInstaller).**
```

Comment:
Do we need the C-Extension or could we leave this out?

Reply:
We don’t need it, no. So we can leave that part out

Reply:
Thought about it more: we could leave it out, but then people wouldn’t be pointed to the auto installer and wouldn't install the extension. So instead I added a comment about standalone integration but still mentioned the installer as the preferred way.